homeassistant/components/wemo/strings.json

```diff
@@ -10,6 +10,22 @@
       "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
     }
   },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "enable_subscription": "Subscribe to device local push updates",
```

Comment:
Instead of giving users the option, I wonder if we can do a similar approach as Shelly does. They wait up to X time for a push update, if not received, do a poll. That way not every user has to learn about polling/networking etc but we have a high change of it working.

Reply:
I may not be fully understanding how Shelly works. pywemo has code to [detect if the subscription is active](https://github.com/pywemo/pywemo/blob/bec12ac573870b208b823078e010ade9888de3a2/pywemo/subscribe.py#L563), and if so the [DataUpdateController disables polling](https://github.com/home-assistant/core/blob/496260561355c9b4f6d7c9fc78638dda88506b8e/homeassistant/components/wemo/wemo_device.py#L107).

Some users have mentioned that the subscription to the push update itself is causing issues for their devices. The subscriptions are working on their device, and then at some point the devices become unresponsive. I'm unable to reproduce that on my devices, but disabling the subscription logic seems to solve the issue those users are having. This option is about giving those users a choice to disable the push subscriptions all-together, which is something I don't think we can detect automatically.